### DISCUSSION TITLE
Design rant.

### DISCUSSION THREAD

**Discussion by danny3man** [fluidd-core/fluidd#1465] [Ideas]
My idea for today is to change the icon for the emergency stop button....
For the first time i opened fluidd on my phone and i saw the red exclamation mark, and thought is a notification particularly one of those "emergency " notifications like "rpi is undervolted"... and i clicked, and just canceled a 5 hour job at 3h30m.
Whoever decided to use exclamation mark for the "emergency stop", f-you very much and to many failed prints and projects!
And also why would you choose that location for it.... duck you again, and me for not being careful enough with bad designs.
![ddd](https://github.com/fluidd-core/fluidd/assets/152693653/b252ae67-2b56-4876-87d2-8508d4769728)
![warning](https://github.com/fluidd-core/fluidd/assets/152693653/e92a6173-2a88-44c1-b931-4c246ad84cb0)

**Comment by matmen** [comment]
Hi, thanks for opening this discussion.

What changes are you suggesting? Do you have any specific icon or design changes in mind?
The current icon on mobile is consistent with the e-stop button on desktop.
It's placed in the bottom left so users can reach it easily in case of an emergency,
and don't have to reach across the entire screen to trigger an e-stop.

We do have a setting that will require confirming an emergency stop via a modal to prevent accidentally stopping a print.

**Comment by danny3man** [comment]
My problem was on the phone, where i think the requiring confirmation option should be on by default, considering the small screen an ease of reach, compared to PC.
![sign](https://github.com/user-attachments/assets/4748d618-224e-4838-a919-f31320d377bb)
![sign_blue](https://github.com/user-attachments/assets/357bcd34-3697-4a94-9a1a-e25add1b3cce)
![sign_simnple](https://github.com/user-attachments/assets/05df2bf2-226f-4210-9f71-042b59f126d8)
Some quick SVG's of a classical emergency stop button i think could replace the button.

**Comment by mnemnion** [comment]
I'd like to suggest that whatever icon is used be red and octagonal.  Worldwide, everyone knows that a red octagon means "stop".  OP is also correct that a circle of white arrows is "the" sign for a CNC emergency stop.

**Comment by Felix14-v2** [comment]
> i saw the red exclamation mark, and thought is a notification particularly one of those "emergency " notifications like "rpi is undervolted"... and i clicked

Literally my situation, except in my case printer was idle.

**Comment by henrivdr** [comment]
Same here. On the list of things you only do once.

I vote for the official CNC/machine button, or classic road octagon Stop sign.

**Comment by pedrolamas** [comment]
I've been doing some tests, here's what I got so far:

## Wide screen

<img width="3808" height="336" alt="image" src="https://github.com/user-attachments/assets/662b351a-ec06-4366-ad31-13afb62964cd" />

## Compact screen

<img width="1043" height="1163" alt="image" src="https://github.com/user-attachments/assets/fcb2d685-9a0c-45b9-be38-a79b0147ab5c" />

**Reply by pedrolamas** [reply]
I've now opened a PR on this here: https://github.com/fluidd-core/fluidd/pull/1897

**Reply by pedrolamas** [reply]
This PR has now been merged, hopefully this helps to clear the issues pointed!
